### PR TITLE
Updated MTA Homework

### PR DESCRIPTION
**Pull request by libbycoy**
Here is my updated MTA homework with the updated second function. Thanks for the help.

### CODE REVIEW

File: libby_coy/week_01/01_hw_mta/js/hw-mta.js [hunk 1]
```diff
@@ -43,54 +43,34 @@ Tell the user the number of stops AND the stops IN ORDER that they will pass thr
...
-	// for (var i = 0; i < lineL.length; i++) {
-	// console.log ( lineL[ i ] );
-	// }
+var planTrip = function(startLine, startStation, endLine, endStation) {
+	if (startLine === endLine) {
+		console.log(routeof(startLine, startStation, endStation));
+	} else {
+		console.log(routeOf(startLine, startStation, 'Union Square')) + routeOf(endLine, 'Union Square', endStation);
+	}
+	}
```
Comment (textchimp) [inline libby_coy/week_01/01_hw_mta/js/hw-mta.js]: Just testing the new comment-on-specific-lines-of-code GitHub functionality

File: libby_coy/week_01/01_hw_mta/js/hw-mta.js [hunk 2]
```diff
@@ -43,54 +43,34 @@ Tell the user the number of stops AND the stops IN ORDER that they will pass thr
...
+	var stopNumOff = currentLine.indexOf(end);
+	if (stopNumOn < stopNumOff) {
+		for (var i = stopNumOn; i <= stopNumOff; i++) {
+			console.log(currentLine[i]);
+	  }
+	} else {
+		for (var i = stopNumOn; i >= stopNumOff; i--) {
+			console.log(currentLine[i]);
+		}
+	}
```
Comment (textchimp) [inline libby_coy/week_01/01_hw_mta/js/hw-mta.js]: still testing

File: libby_coy/week_01/01_hw_mta/js/hw-mta.js [hunk 3]
```diff
@@ -43,54 +43,34 @@ Tell the user the number of stops AND the stops IN ORDER that they will pass thr
...
-// my stopNumOn & stopNumOff values?
-
-	for (var i = stopNumOn; i <= stopNumOff; i++) {
-	console.log(lineN[i]);
-// 	return result;
+var subway = {
+	N: ['Times Square', '34th', '28th', '23rd', 'Union Square', '8th'],
+	L: ['8th', '6th', 'Union Square', '3rd', '1st'],
+	6: ['Grand Central', '33rd', '28th', '23rd', 'Union Square', 'Astor Place']
 }
```
Comment (textchimp) [inline libby_coy/week_01/01_hw_mta/js/hw-mta.js]: also still testing

File: libby_coy/week_01/01_hw_mta/js/hw-mta.js [hunk 4]
```diff
@@ -43,54 +43,34 @@ Tell the user the number of stops AND the stops IN ORDER that they will pass thr
...
-var stopNumOff = lineN.indexOf(route.stop2);
-
-// var routeOf = function(route.stop1, route.stop2) {
-// how do I put the below into a variable while still keeping all of
-// my stopNumOn & stopNumOff values?
-
-	for (var i = stopNumOn; i <= stopNumOff; i++) {
-	console.log(lineN[i]);
-// 	return result;
+var subway = {
```
Comment (textchimp) [inline libby_coy/week_01/01_hw_mta/js/hw-mta.js]: more still testing

File: libby_coy/week_01/01_hw_mta/js/hw-mta.js (hunk 1 above)
Comment (textchimp) [inline libby_coy/week_01/01_hw_mta/js/hw-mta.js:74]: Looks like you've got it.

All I would say is that the line in the `else` block is a little less readable than if you used a couple of lines, i.e.

``` javascript
console.log(routeOf(startLine, startStation, 'Union Square'));
console.log(routeOf(endLine, 'Union Square', endStation));
```

This is one extra call to `console.log()`, but it's more readable: you can now compare each call to `routeOf()` and see how they're similar and how they're different.

I would also recommend that you get used to adding a line or two of comments to your code, anywhere that you're doing something that is not completely obvious; so the above few lines of code are a good example:

``` javascript
// if the trip is between two stations that are not on the same line, we simply treat the 
// multiple line trip as two single line trips, with the exchange station Union square as:
// - the last station of the starting line, and
// - the first station of the end line
console.log(routeOf(startLine, startStation, 'Union Square'));
console.log(routeOf(endLine, 'Union Square', endStation));
```

The above example of comments is good in that it explains the trick of a line-swtiching trip as a combination of same-line trips, but on the other hand, it's getting almost too close to just saying what the code is obviously doing. Avoid overly obvious comments such as in this hypothetical code:

``` javascript
// test if validName is true
if (validName === true) {
```

**Comment by textchimp** [comment]
Nice one on getting MTA working. 

Just a point on indentation in your code: you seem to be using 8 spaces for one level of indentation, which is too much; with a few levels of indentation (i.e. nested loops or `if` statements), your code will soon get pushed too far to the right and start wrapping around. I would recommend 2 spaces, or 4 spaces maximum.

File: libby_coy/week_02/02_homework_02/index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <title>Jenn</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="font-awesome-4.6.3/css/font-awesome.min.css">
+
+
+  </head>
+
+  <body class="index-body">
+    <header>
+        <a href="./index.html"><h1 class="index"><i>JENN <span style="color: #BF6658" hover="#109675">SCHIFFER</span></i></h1></a>
```
Comment (textchimp) [inline libby_coy/week_02/02_homework_02/index.html:15]: There's no need to use that leading `./` for `href` or `src` paths when the file is in the same folder; it's customary to just write `href="index.html"`.

**Comment by textchimp** [comment]
Your fansite looks great. I wish there was some way to show it to Jenn Schiffer without creeping her out.

**Comment by libbycoy** [comment]
Thanks heaps. Haha would probably be creepy!

**Pull request merged**